mmv1/third_party/terraform/services/cloudfunctions/resource_cloudfunctions_function.go

```diff
@@ -490,6 +490,11 @@ func ResourceCloudFunctionsFunction() *schema.Resource {
 				Computed:    true,
 				Description: `Describes the current stage of a deployment.`,
 			},
+			"version_id": {
+				Type:        schema.TypeInt,
```

Comment:
It looks like this is a `string (int64)` in the API, so it should be represented as a string in Terraform for consistency. TypeInt can't reliably represent int64 values.

Reply:
Sure, updated!